### DISCUSSION TITLE
Antlers Overhaul Coming Soon?

### DISCUSSION THREAD

**Discussion by digitalam025** [statamic/cms#4877] [General]
Hi! I'm really curious about how the antler's rewriting (#4257) is coming along? Is it gonna be released to an update super duper soon? 🤞 🤞 I thought it was gonna be added in early November 😮

**Comment by duncanmcclean** [comment]
I think it's still a few months away until it's released... not sure if they have an exact date for it yet. It's being built by a contributor, not by the Core Team.

**Reply by digitalam025** [reply]
Aw man... But thanks tho! :)
